frontend21/scss/amp-dev.scss

```diff
@@ -1,5 +1,55 @@
-@import '../frontend/scss/_functions.scss';
-@import '../frontend/scss/_mixins.scss';
-@import '../frontend/scss/_variables.scss';
+@import '../../frontend/scss/_functions.scss';
+@import '../../frontend/scss/_mixins.scss';
+@import '../../frontend/scss/_variables.scss';
+@import '../../frontend/scss/components/atoms/_text.scss';
+@import '../../frontend/scss/components/atoms/_color.scss';
 
+@import 'components/section.scss';
 @import 'components/text-media.scss';
+@import 'components/text-block.scss';
+@import 'components/button.scss';
+@import 'components/pie-chart.scss';
+
+main {
+  padding: 0;
+  width: 100%;
+  max-width: 1920px;
+  margin: auto;
+
+  @media (min-width: 768px) {
+    display: grid;
```

Comment:
I think it would make sense to use grid layout for mobile as well; then you also wouldn't have to deal with manual paddings on mobile but could also simply rely on `grid-gap`.